### PR TITLE
chore(RHTAPWATCH-1169): Reduce permissions in the appstudio-env tier

### PR DESCRIPTION
**Pull request by hmariset**
Restrict the permissions the service account in ephemeral namespace has in creating, updating and patching the Routes and Ingresses resources.

Jira-Url: https://issues.redhat.com/browse/RHTAPWATCH-1169

### CODE REVIEW

File: components/sandbox/tiers/staging/appstudio-env/tiertemplate-appstudio-env-admin-37575401-37575401.yaml [hunk 1]
```diff
@@ -12,7 +12,8 @@ spec:
   template:
     apiVersion: template.openshift.io/v1
     kind: Template
-    metadata: {}
+    metadata:
+      creationTimestamp: null
```
Comment (hugares) [inline components/sandbox/tiers/staging/appstudio-env/tiertemplate-appstudio-env-admin-37575401-37575401.yaml:16]: All these changes relate to creationTimestamp should not be here. You need to generate the tiers using  @dperaza4dustbit branch: https://github.com/kubesaw/ksctl/pull/50 of ksctl

File: components/sandbox/tiers/staging/appstudio-env/kustomization.yaml [hunk 2]
```diff
@@ -7,6 +7,7 @@ kind: Kustomization
 resources:
 - nstemplatetier-appstudio-env.yaml
 - tiertemplate-appstudio-env-admin-37575401-37575401.yaml
+- tiertemplate-appstudio-env-clusterresources-1854252301-1854252301.yaml
 - tiertemplate-appstudio-env-clusterresources-266544570-266544570.yaml
```
Comment (amisstea) [inline components/sandbox/tiers/staging/appstudio-env/kustomization.yaml:11]: @dperaza4dustbit is it safe to remove this older tiertemplate along with these changes?

File: components/sandbox/tiers/staging/appstudio-env/tiertemplate-appstudio-env-admin-37575401-37575401.yaml (hunk 1 above)
Comment (hugares) [reply components/sandbox/tiers/staging/appstudio-env/tiertemplate-appstudio-env-admin-37575401-37575401.yaml]: Done, I re-generated the tiers using that version of ksctl

File: components/sandbox/tiers/staging/appstudio-env/kustomization.yaml (hunk 2 above)
Comment (hugares) [reply components/sandbox/tiers/staging/appstudio-env/kustomization.yaml:11]: I asked that already in another PR I did and the answer was leave it there for now, they will do clean up eventually

**Review by amisstea — APPROVED** [review]

**Review by hugares — APPROVED** [review]
/lgtm

**Pull request merged**